api/tests/e2e/e2e_suite_test.go

```diff
@@ -138,9 +140,9 @@ func deleteSubnamespace(parent, name string) {
 		},
 	}
 	err := k8sClient.Delete(ctx, &anchor)
-	Expect(err).NotTo(HaveOccurred())
+	ExpectWithOffset(2, err).NotTo(HaveOccurred())
```

Comment:
I am a bit skeptical about using offsets as you cannot really know the stack depth the utility function usage. I wish failures also reported a stack trace, similar to exceptions in Java, unfortunately golang errors does not support that out of the box

Reply:
Me too :). I had to adjust these while I was driving this, since the ginkgo failures were really cryptic and low level. Not sure what is the best way of handling this.

Reply:
This was new to me as well.. so just so I'm understanding it right, using `ExpectWithOffset` should show the _actual test code line_ that failed instead of the LOC of this helper function? I share how y'all feel about this feeling brittle, but I don't know a better way either.

Maybe @matt-royal has Gomega thoughts as well. 🤔

Reply:
I agree that offsets are brittle. Also, when I've experimented with them I didn't find the resulting stack traces to be any clearer.